app/views/layouts/application.html.haml

```diff
@@ -72,5 +74,7 @@
         $('.ga-client-id').val(clientId);
       });
 
+  -# govuk-frontend (Design System)
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3387/files#r440061157)